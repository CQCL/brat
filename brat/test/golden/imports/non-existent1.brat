import lib (x, a)

main :: Nat
main = x