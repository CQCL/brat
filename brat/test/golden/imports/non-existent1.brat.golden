Error in non-existent1 on line 1:
import lib (x, a)
               ^

  Symbol `a` not found in `lib`

